packages/ozone/src/communication-service/template.ts

```diff
@@ -0,0 +1,73 @@
+import Database from '../db'
+import { Selectable } from 'kysely'
+import { CommunicationTemplate } from '../db/schema/communication_template'
+import { CommunicationTemplateView } from '../lexicon/types/com/atproto/admin/defs'
+
+export type CommunicationTemplateServiceCreator = (
+  db: Database,
+) => CommunicationTemplateService
+
+export class CommunicationTemplateService {
+  constructor(public db: Database) {}
+
+  static creator() {
+    return (db: Database) => new CommunicationTemplateService(db)
+  }
+
+  async list(): Promise<Selectable<CommunicationTemplate>[]> {
+    const list = await this.db.db
+      .selectFrom('communication_template')
+      .selectAll()
+      .execute()
+
+    return list
+  }
+
+  async create(
+    template: Omit<Selectable<CommunicationTemplate>, 'id'>,
+  ): Promise<Selectable<CommunicationTemplate>> {
+    const newTemplate = await this.db.db
+      .insertInto('communication_template')
+      .values(template)
```

Comment:
There might be some value in being explicit about which fields are passed into `values()`, and same goes with `set()` in the update method.  I notice that the request body is spread into `template` within the handlers, and it may specify some columns that don't exist or internal columns that we don't want to allow being set.

Reply:
yeah that makes sense!